angular-chart.js

```diff
@@ -146,6 +147,11 @@
 
             chart = createChart(chartType, scope, elem);
           }
+
+          angular.element(window).on('resize', function () {
```

Comment:
It would be better to listen on the event only if the `autoresize` option has been set.